package/packer/vagrant/vagrant.json

```diff
@@ -1,7 +1,7 @@
 {
   "variables": {
     "password": "rancher",
-    "version": "v0.4.0",
+    "version": "v0.5.0",
     "checksum": "e3a0b0f22ac492fc76363a0319c733f392c6ce829825f3dc76cd72a45da384ca"
```

Comment:
Please update the checksum in the Vagrant template.

Reply:
Ah, right. Updated.

Reply:
Have tested GCP and Vagrant. Assuming AWS will work fine as nothing has changed there, but I don't have an active account with them.